README.md

```diff
@@ -16,14 +16,9 @@ None.
 
 ## Role Variables ##
 
-None.
-
-<!--
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| optional_variable | Describe its purpose. | `default_value` | No |
-| required_variable | Describe its purpose. | n/a | Yes |
--->
+| clamav_cron_frequency | Set the cronjob frequency of clamscan. Must be one of : `hourly`,`daily`,`weekly` or `monthly` | `weelky` | No |
```

Comment:
Fix typo and tweak language:
```suggestion
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`,`daily`,`weekly`, or `monthly`. | `weekly` | No |
```

Reply:
@jsf9k Would you tweak your suggestion to add spaces after the other commas in the list?

Reply:
Thanks to @mcdonnnj and his eagle eyes!